qcelemental/models/molecule.py

```diff
@@ -228,6 +228,13 @@ def json(self, *args, **kwargs):
 
 ### Non-Pydantic API functions
 
+    def measure(self, measurements, degrees=True):
```

Comment:
Are degrees really the default people expect for quantum chemistry applications? Just a side thought is all

Reply:
I think so, happy to take other comments here however,

Reply:
user-facing is usually degrees, I think. coders would rather convert than deal with confused gen chem students running labs and touting vespr. :-)